presto-mongodb/src/main/java/com/facebook/presto/mongodb/ObjectIdFunctions.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.mongodb;
+
+import com.facebook.presto.operator.Description;
+import com.facebook.presto.operator.scalar.ScalarFunction;
+import com.facebook.presto.operator.scalar.ScalarOperator;
+import com.facebook.presto.spi.type.StandardTypes;
+import com.facebook.presto.type.SqlType;
+import com.google.common.base.CharMatcher;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+import org.bson.types.ObjectId;
+
+import static com.facebook.presto.metadata.OperatorType.EQUAL;
+import static com.facebook.presto.metadata.OperatorType.NOT_EQUAL;
+
+public class ObjectIdFunctions
```

Comment:
FYI, I merged the changes in this PR with tag 0.122 and attempted to run on my MongoDB instance.  The server threw:
```
2015-10-14T16:09:54.669-0400    ERROR   main    com.facebook.presto.server.PrestoServer [GREATER_THAN, BETWEEN, LESS_THAN_OR_EQUAL, GREATER_THAN_OR_EQUAL, HASH_CODE, LESS_THAN] missing for ObjectId
java.lang.IllegalStateException: [GREATER_THAN, BETWEEN, LESS_THAN_OR_EQUAL, GREATER_THAN_OR_EQUAL, HASH_CODE, LESS_THAN] missing for ObjectId
        at com.facebook.presto.metadata.MetadataManager.verifyComparableOrderableContract(MetadataManager.java:227)
        at com.facebook.presto.server.PluginManager.loadPlugins(PluginManager.java:157)
        at com.facebook.presto.server.PrestoServer.run(PrestoServer.java:111)
        at com.facebook.presto.server.PrestoServer.main(PrestoServer.java:62)
```
I added the missing `ObjectId` operators to `ObjectIdFunctions` and now have a running presto server.